include/veriblock/pop/blockchain/pop/fork_resolution.hpp

```diff
@@ -552,6 +552,7 @@ struct PopAwareForkResolutionComparator {
       }
 
       VBK_LOG_DEBUG("Candidate contains VALID commands, chain B wins");
+      ed_.overrideTip(candidate);
```

Comment:
Why did you make this change?

Reply:
Because I want activateBestChain to be consistent. It should always switch to the winner.